RecFCCeeCalorimeter/CMakeLists.txt

```diff
@@ -2,6 +2,13 @@
 # Package: RecFCCeeCalorimeter
 ################################################################################
 
+# ONNX DEPENDENCIES
```

Comment:
can you put `find_package` to the top CMake file? Also update list of dependencies in the README.

Reply:
fixed in latest commit